DOCS-2854: Setting up DKIM email authentication (#1938)
MultiSafepay/docs: content/my-account/manage-account/setting-up-DKIM.md

---
title: "Setting up DKIM email authentication"
weight: 12
meta_title: "Setting up DKIM email authentication - MultiSafepay Docs"
read_more: "."
url: '/my-account/setting-up-dkim/'
---

> MultiSafepay supports DomainKeys Identified Mail (DKIM) email authentication on all emails that we send. DKIM allows email servers to verify that emails they receive are actually from that domain and have not been altered or forged.

Reviewer: ```suggestion
MultiSafepay supports DomainKeys Identified Mail (DKIM) email authentication for all emails that we send. DKIM lets email servers verify that received emails actually came from the specified domain, and haven't been altered or forged.
```

Reviewer: Official technical writer usage:
Allows= Relates to permissions
Lets= Makes it possible to do something
Enables= Turns a feature on

> DKIM authenticates emails through a pair of cryptographic keys. We use the private key to encrypt the email content and generate a DKIM signature before sending. You set the public key in a TXT record through your hosting provider, domain registrar, or DNS provider. This allows receiving email servers to access the public key and use it to verify the integrity of the email and the authenticity of the sender.

Reviewer: ```suggestion
DKIM authenticates emails using two cryptographic keys:  

 - Private key: MultiSafepay uses this to encrypt the email content and generate a DKIM signature before sending. 
 - Public key: You set this in a TXT record through your hosting provider, domain registrar, or DNS provider. Receiving email servers can access and use it to verify the integrity of the email and the authenticity of the sender.
```

Author: I've included this change but left the reference to the **pair** of keys as this is a security term and they are intrinsically linked. If we say: **two** cryptographic keys, then some readers might interpret that as two pairs of keys.

## Adding TXT records

> We use Mandrill in addition to our own mail servers to send emails. So two TXT records need to be added.

Reviewer: ```suggestion
In addition to our own mail servers, we use Mandrill to send emails. This means you need to add **two** TXT records.
```

### MultiSafepay email servers

> 1. Add a TXT record named: `msp-2021._domainkey.{your domain}`, where `{your domain}` contains your domain.
> {{< br >}}For example: `msp-2021._domainkey.myamazingwebshop.com`

Reviewer: ```suggestion
1. Add a TXT record named `msp-2021._domainkey.{your domain}`, e.g. `msp-2021._domainkey.myamazingwebshop.com`.
```

2. Add the following content to your TXT record:
    ```
    v=DKIM1; h=sha256; k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4pizvsOWTWxtcxthGR0k4rEcGsJH4hRy1fpoUAs3fUi0yMkygwsYUCtFLQY2TwrOtfPfaZ/2bPKXwyjC4kg93zFvSJTIQtQiFfKNT2aDtnDmZRwII4+s2k7+LHn4V/SjIxEBylN3Rt0g4iVlkZzgncEXeVksXj5eux8uDAUeZxj0Fp8PWSkxsBNVaJFb5sfR+c5piJ+8RmlqYUf7w/gXOW8mChC509//V9dfMaV39b7WoEf/JRw9KGM69C3hIdtb7cVKD/B6VxQIq3z1DCAcmSCXpcaXUaFbVaF4u/vEi+3v5DdPtDl/0rOy2NUFNL5XULW8OxdofzUbdL9SWN/IbwIDAQAB;
    ```
You have successfully added a TXT record for MultiSafepay's email servers.

### Mandrill

> 1. Add a TXT record named: `mandrill._domainkey.{your domain}`, where `{your domain}` contains your website domain.
> {{< br >}}For example: `mandrill._domainkey.myamazingwebshop.com`

Reviewer: ```suggestion
1. Add a TXT record named `mandrill._domainkey.{your domain}`, e.g. `mandrill._domainkey.myamazingwebshop.com`.
```
